pyemma/msm/api.py

```diff
@@ -1288,6 +1292,163 @@ def bayesian_hidden_markov_model(dtrajs, nstates, lag, nsamples=100, reversible=
                                   dt_traj=dt_traj, conf=conf, store_hidden=store_hidden, show_progress=show_progress)
     return bhmsm_estimator.estimate(dtrajs)
 
+def estimate_augmented_markov_model(dtrajs, ftrajs, m, w, lag, 
```

Comment:
I suggest the order ``dtrajs, ftrajs, lag, m, w`` (first simulation data and parameter, then experimental constraints)